src/components/DeleteButton.jsx

```diff
@@ -0,0 +1,34 @@
+import styled from "@emotion/styled";
+import DeleteIcon from "@assets/icons/delete";
+
+const BUTTON_SIZE = {
+  s: "1.6rem",
+  m: "2rem",
+  l: "2.4rem",
+};
+
+const DeleteButton = ({ onClick, size = "s" }) => {
+  return (
+    <DeleteButtonWrapper onClick={onClick} size={BUTTON_SIZE[size]}>
```

Comment:
❗️ 수정요청
이런 클릭 가능한 요소의 경우 버튼을 사용해주세요. 더 적절한 태그가 있는데 div 태그에 onClick 을 주게되면 button에게 기대하는 동작을 제대로 수행하지 못합니다!

```suggestion
    <DeleteButtonWrapper type="button" onClick={onClick} size={BUTTON_SIZE[size]}>
```

버튼으로 바꿔주시고 type을 명시하시는 것을 추천드려요~